src/Wallabag/CoreBundle/Resources/translations/messages.en.yml

```diff
@@ -103,6 +103,7 @@ config:
             archive: 'Archived'
             all: 'All'
         feed_limit: 'Number of items in the feed'
+        feed_use_source: 'Return source url by default into ATOM feed instead of Wallabag uri'
```

Comment:
I am not able to provide translations for all languages. Hope it will be ok for you.

Reply:
It's fine it'll be handle by translators using Weblate